docs/meshstack.identity-lookup.md

```diff
@@ -166,7 +168,34 @@ let exampleWithSchmaExtension
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-To use a non-standard attribute as
+To use a non-standard attribute as euid
```

Comment:
d: it would be helpful if this would say what the standard attribute is, e.g. when `emailAttributeUpdate` is a `None AzureEmailAttributeUpdate`, what is the default attribute used?

Reply:
Added defaults where defaults were there. Removed the `non-standard` text for those that didn't have a default.